fly_manager.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
```

Comment:
I'll clean most of this up once I get approval that the steps in the README work